architecture/topics/haproxy_template_router.adoc

```diff
@@ -27,4 +27,106 @@ The following diagram illustrates how data flows from the master through the
 plug-in and finally into an HAProxy configuration:
 
 .HAProxy Router Data Flow
-image::router_model.png[HAProxy Router Data Flow]
\ No newline at end of file
+image::router_model.png[HAProxy Router Data Flow]
+
+[[haproxy-metrics]]
+=== HAProxy Template Router Metrics
+
+The HAProxy router, by default, exposes or publishes metrics in
+link:https://prometheus.io/docs/concepts/data_model/[Prometheus format]
+for consumption by external metrics collection and aggregation systems (e.g. Prometheus, statsd).
+The router can be
+xref:../../install_config/router/default_haproxy_router.adoc#exposing-the-router-metrics[confiugred]
+to provide
+link:https://cbonte.github.io/haproxy-dconv/1.5/configuration.html#9[HAProxy CSV format] metrics, or
+provide no router metrics at all.
+
+Prometheus format data is the default data format for metrics in {product-title}.
```

Comment:
I'm not sure we need to say this.

Or can we integrate it with the previous paragraph.

Reply:
Dropped line